site/siteConfig.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import siteConfig from 'terra-site/lib/config/site.config';
+import Home from './Home';
+
+const homeLink = [{
+  path: '/site/home',
+  text: 'Home',
+  component: Home,
+  isStatic: true,
```

Comment:
- The `text` key sets the [NavTab title](https://github.com/cerner/terra-site/blob/dynamic-site/src/app/common/nav-tabs/NavTabs.jsx#L44) in the ApplicationHeader.
- The `component` key tells the routeConfiguration function in the configureApp component what to render with when hitting a route. [See here](https://github.com/cerner/terra-site/blob/dynamic-site/src/app/configureApp.jsx#L110). This leaves component and test rendering open to customized, however, any custom component will receive [these props](https://github.com/cerner/terra-site/blob/dynamic-site/src/app/configureApp.jsx#L114)
- the `isStatic` key indicate if the routeConfiguration function should generate a ComponentsMenu